readthedocs/projects/models.py

```diff
@@ -481,11 +481,11 @@ def full_dash_path(self, version=LATEST):
 
     def full_json_path(self, version=LATEST):
         """The path to the build json docs in the project."""
-        if 'sphinx' in self.documentation_type:
-            return os.path.join(self.conf_dir(version), '_build', 'json')
-
-        if 'mkdocs' in self.documentation_type:
-            return os.path.join(self.checkout_path(version), '_build', 'json')
+        json_path = os.path.join(
+            self.conf_dir(version),
+            '_build', 'json'
+        )
+        return json_path
```

Comment:
So, this is called from `process_all_json_files`

https://github.com/rtfd/readthedocs.org/blob/f06271b698d26734bf5024590948d41b1613b2e3/readthedocs/search/parse_json.py#L21-L26

And is used in `os.walk`, if the path doesn't exists (mkdocs for example), it doesn't raise an error, it just returns an _empty_ iterator.